tavern_server/src/db_tables.sql

```diff
@@ -0,0 +1,861 @@
+--enum types
```

Comment:
Why is this file part of the pull request? It's already part of the other one, right?

Reply:
I'm not actually sure, this file shouldn't contain any modifications at all that aren't in TVRN-13. Removing this file from the pull request now.